index.js

```diff
@@ -25,6 +25,19 @@ export const config = {
 const usePreset = () => atom.config.get('linter-stylelint.usePreset');
 const presetConfig = () => atom.config.get('linter-stylelint.presetConfig');
 
+function getRange(data) {
```

Comment:
Huh, looks like although it is in the dependencies, it's not currently being required. I mean `const helpers = require('atom-linter');`.